app/mailers/ban_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class BanMailer < ApplicationMailer
+  default from: 'The Odin Project <contact@theodinproject.com>'
+
+  def send_ban_email_to(user)
```

Comment:
Nit: Because this is a user related email, I think it should live in the `UserMailer` along side the welcome email.